tests/certbot-boulder-integration.sh

```diff
@@ -527,3 +529,62 @@ if [ "${BOULDER_INTEGRATION:-v1}" = "v2" ]; then
 fi
 
 coverage report --fail-under 64 --include 'certbot/*' --show-missing
+
+# Test OCSP status
+
+## OCSP 1: Check staled certificate OCSP status
+pushd ./tests/letstest/testdata
+
+OUT=`common certificates --config-dir sample-config`
+TEST_CERTS=`echo "$OUT" | grep TEST_CERT | wc -l`
+EXPIRED=`echo "$OUT" | grep EXPIRED | wc -l`
+REVOKED=`echo "$OUT" | grep REVOKED | wc -l`
+
+if [ "$TEST_CERTS" != 2 ] ; then
+    echo "Did not find two test certs as expected ($TEST_CERTS)"
+    exit 1
+fi
+
+if [ "$EXPIRED" != 2 ] ; then
+    echo "Did not find two test certs as expected ($EXPIRED)"
+    exit 1
+fi
+
+if [ "$REVOKED" != 0 ] ; then
```

Comment:
I think this will fail in our nightly tests where we run integration tests with our oldest dependencies. When doing that, we will be shelling out to openssl and one of these certificates will be listed as revoked.

I personally think this minor inconsistency is fine and we should remove this check.

Reply:
Indeed. We test the revoke on a live certificate just after. I remove that.